.github/workflows/publish-de-image.yaml

```diff
@@ -21,17 +21,20 @@ on:
     #   dest_image: Destination image in GHCR (e.g. ghcr.io/radius-project/deployment-engine)
     #   tag: Tag for the image (e.g. 1.0)
 
+run-name: "(Deployment Engine) Publish Image - Tag: ${{ github.event.client_payload.tag || github.event.inputs.tag }}"
```

Comment:
is `run-name` going to cause problems? or is it okay as long it's not a required check